templates/show-worklist.html

```diff
@@ -26,6 +28,10 @@
 	    input_by_username.list = Array.from(new Set(worklists.map(worklist => worklist.created_by)));
 
 	    setTimeout(worker, 50000);
+
+        $(document).arrive(".fixed-table-pagination", function() {
+    		restoreState();
+		});
```

Comment:
Indentation looks off in the `});` line, though maybe this is just GitHub messing up the formatting.

Reply:
Yeah, indentation is correct in my editor. This is GitHub messing it up.